test/specs/templates.e2e.ts

```diff
@@ -148,56 +362,54 @@ describe('Templates', async () => {
     expect(outputPanelText).toContain('Created  auraEvent1  AuraDefinitionBundle');
   });
 
-  // Apex Class
-  step('Create an Apex Class', async () => {
-    // Using the Command palette, run SFDX: Create Apex Class.
+  // Aura Interface
+  step('Create an Aura Interface', async () => {
+    // Clear the output panel, then use the Command palette to run, "SFDX: Create Aura Interface".
     const workbench = await browser.getWorkbench();
-    const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 1);
+    const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Aura Interface', 1);
 
-    // Set the name of the new component to ApexClass1.
-    await inputBox.setText('ApexClass1');
+    // Set the name of the new component to AuraInterface1.
```

Comment:
```suggestion
    // Set the name of the new interface to AuraInterface1.
```

Reply:
Darn, too much copy-pasting.  This has been fixed.